docs:  improvements (#283)
nuxt/image: docs/content/en/3.api/2.$img.md

---
menuTitle: $img
title: $img

> description: 'Using $img programmatically'

Reviewer: The purpose is exactly generating URLs :D (not necessarily transform) and usage is practically possible within a `programmatic` context (anywhere in source code)

category: API
---

> Sometimes it's necessary to apply image transformation programatically. You can use `$img` utility in context to generate image URLs for any purpose not possible with `<nuxt-img>` and `<nuxt-picture>` components.

Reviewer 2: ```suggestion
Sometimes it's necessary to apply image transformations programatically. You can use the `$img` utility available in the Nuxt context to generate image URLs if the `<nuxt-img>` and `<nuxt-picture>` components aren't suitable.
```
